src/pages/javascript/with/index.md

```diff
@@ -3,13 +3,46 @@ title: With
 ---
 ## With
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/javascript/with/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+JavaScript's `with` statement is a shorthand way for editing several properties on one object. Most developers discourage usage of `with`, and you are best not using this keyword.
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+**Note**: `"strict mode"` forbids usage of `with`.
+
+
+### Syntax
+```syntax
+with (expression)
+  statement
+```
+
+
+### Example Usage
+In JavaScript, you can individually modify an object's properties like below:
+```javascript
+earth.land.tree.kind = 'apple';
+earth.land.tree.height = 4;
+```
+`with` gives you a shorthand to modify the properties on an object:
+```javascript
+with (earth.land.tree) {
+  kind = 'apple';
+  height = 4;
+}
+```
+
+
+### Alternatives
+Instead of using `with`, a much better approach is to assign the object to a variable, and then modify the variable's properties.
+```javascript
+let tree = earth.land.tree;
+tree.kind = true;
+tree.height = true;
```

Comment:
From the changes above, this example would have to change or create a new example because the one I referenced above only goes one object level deep.